.gitignore

```diff
@@ -5,4 +5,4 @@ projects/Mallard/.vscode/.react/*
 projects/backend/.cache/*
 projects/backend/dist*
 projects/aws/cloudformation.yaml
-.DS_Store
```

Comment:
What's the harm in leaving DS_Store in the .gitignore?

Reply:
I left the one on line 2....